meson.build

```diff
@@ -8,7 +8,7 @@ project(
     default_options: [
         'cpp_std=c++17',
         'buildtype=release',
-        'default_library=static',
+        'default_library=shared',
```

Comment:
This can simply be removed if you like, as that's the default builtin value.
OTOH some people would say explicit is better.

Either approach is fine and does the same thing.

Reply:
(removed)